net-dns/dnscrypt-proxy/files/dnscrypt-proxy-2.0.0_fix_systemd_service.patch

```diff
@@ -0,0 +1,8 @@
+--- a/systemd/dnscrypt-proxy.service	2018-02-09 11:04:36.424058868 +0100
++++ b/systemd/dnscrypt-proxy.service	2018-02-09 11:05:51.209593944 +0100
+@@ -15,4 +15,4 @@
+ NonBlocking=true
+ 
+ # Change this
+-ExecStart=/opt/dnscrypt-proxy/dnscrypt-proxy
++ExecStart=/usr/sbin/dnscrypt-proxy --config /etc/dnscrypt-proxy/dnscrypt-proxy.toml
```

Comment:
not sure if using a patch for a single line is justified. Simple sed call will do.
Also this will break prefix, need to use _${EROOT}_

Reply:
I have a different approach. Generally speaking, I don't like sed(s) on original sources because if the sources change I prefer to have a failing patch instead a silently not working sed. Imho